apps/bajour/components/bajour/basel-briefing.tsx

```diff
@@ -0,0 +1,397 @@
+import {Button, styled} from '@mui/material'
+import {BuilderTeaserProps, Image, TeaserWrapper} from '@wepublish/website'
+
+import shouldDisplayBaselBriefing from '../../utils/should-display-basel-briefing'
+import {NextWepublishLink} from '../should-be-website-builder/next-wepublish-link'
+import {fluidTypography} from '../website-builder-overwrites/blocks/teaser-overwrite.style'
+import BaselBg from './basel.jpg'
+import FasnachtBg from './fasnacht.jpg'
+import FcbBg from './fcb.jpg'
+import {BriefingType} from './is-briefing'
+
+type Property = {
+  key: string
+  value: string
+}
+
+type BaselBriefingProps = Omit<BuilderTeaserProps, 'teaser'> & {
+  // has to be any as we cannot import CustomTeaser from @wepublish/website/api
```

Comment:
why?

Reply:
![Uploading Screenshot 2024-02-15 at 11.25.31.png…]()

Reply:
Image is not working. But if it's about linting, read the error message it tells you what to do.

Reply:
it's about importing from @wepublish/website/api, which is not allowed and I should import from @wepublish/website instead. But @wepublish/website doesn't export CustomTeaser

Reply:
But it does. `ApiV1.CustomTeaser` checkout other places that do this.